.github/workflows/breaking-change-alert.yaml

```diff
@@ -155,6 +155,6 @@ jobs:
                 }
               ]
             }
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.NV_SLACK_BREAKING_CHANGE_NOTIFIER_APP }}
-          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+          payload-templated: true
+          webhook: ${{ secrets.NV_SLACK_BREAKING_CHANGE_NOTIFIER_APP }}
+          webhook-type: incoming-webhook
```

Comment:
v2 removed the use of environment variables for configuring the webhook URl and type, in favor of direct workflow inputs:

* "Technique 3: Slack incoming webhook" at https://github.com/slackapi/slack-github-action/releases/tag/v2.0.0
* https://github.com/slackapi/slack-github-action/issues/372